x/concentrated-liquidity/math/math.go

```diff
@@ -181,7 +181,7 @@ func GetLiquidityFromAmounts(sqrtPrice, sqrtPriceA, sqrtPriceB sdk.Dec, amount0,
 	if sqrtPrice.LTE(sqrtPriceA) {
 		// If the current price is less than or equal to the lower tick, then we use the liquidity0 formula.
 		liquidity = Liquidity0(amount0, sqrtPriceA, sqrtPriceB)
-	} else if sqrtPrice.LTE(sqrtPriceB) {
+	} else if sqrtPrice.LT(sqrtPriceB) {
 		// If the current price is between the lower and upper ticks (non-inclusive of the lower tick but inclusive of the upper tick),
```

Comment:
Does the comment here need to change now? This is non inclusive on both ends now right?

Reply:
Good catch, fixed! 2945e889865dbe993ff061099a5cc3dbc41f0fec